balena/types/models.py

```diff
@@ -335,7 +336,7 @@ class TypeDevice(TypedDict):
     is_running__release: OptionalNavigationResource["ReleaseType"]
     should_be_running__release: OptionalNavigationResource["ReleaseType"]
     is_managed_by__service_instance: OptionalNavigationResource[ServiceInstanceType]
-    should_be_managed_by__supervisor_release: OptionalNavigationResource[SupervisorReleaseType]
+    should_be_managed_by__release: OptionalNavigationResource[SupervisorReleaseType]
```

Comment:
Let's also add the synonymous form `release.should_manage__device` in the same PR so that it includes both the normal fact type as well as its synonymous form.